README.md

```diff
@@ -415,7 +423,7 @@ spec:
     key: prometheusLoginCredentials
 ```
 
-**Note:** The KeptnMetricsProvider is a new resource in KLT 0.7.0.
+> **Note:** The KeptnMetricsProvider is a new resource in KLT 0.7.0.
```

Comment:
Please, have a look at this PR for some suggestions about notes https://github.com/keptn/lifecycle-toolkit/pull/962